tools/deployment/pytorch2onnx.py

```diff
@@ -89,79 +89,74 @@ def pytorch2onnx(config_path,
             you may have to build mmcv with ONNXRuntime from source.')
 
     if do_simplify:
-        from mmdet import digit_version
         import onnxsim
 
+        from mmdet import digit_version
+
         min_required_version = '0.3.0'
         assert digit_version(onnxsim.__version__) >= digit_version(
             min_required_version
         ), f'Requires to install onnx-simplify>={min_required_version}'
 
-        input_dic = {'input': one_img.detach().cpu().numpy()}
+        input_dic = {'input': img_list[0].detach().cpu().numpy()}
         onnxsim.simplify(
             output_file, input_data=input_dic, custom_lib=ort_custom_op_path)
     print(f'Successfully exported ONNX model: {output_file}')
 
     if verify:
-        from mmdet.core import get_classes, bbox2result
-        from mmdet.apis import show_result_pyplot
-
-        model.CLASSES = get_classes(dataset)
-        num_classes = len(model.CLASSES)
         # check by onnx
         onnx_model = onnx.load(output_file)
         onnx.checker.check_model(onnx_model)
+
+        # wrap onnx model
+        onnx_model = ONNXRuntimeDetector(output_file, model.CLASSES, 0)
         if dynamic_export:
             # scale up to test dynamic shape
             h, w = [int((_ * 1.5) // 32 * 32) for _ in input_shape[2:]]
+            h, w = min(1344, h), min(1344, w)
             input_config['input_shape'] = (1, 3, h, w)
-        if test_img is not None:
-            input_config['input_path'] = test_img
+
+        if test_img is None:
+            input_config['input_path'] = input_img
+
+        # prepare input once again
         one_img, one_meta = preprocess_example_input(input_config)
-        tensor_data = [one_img]
+        img_list, img_meta_list = [one_img], [[one_meta]]
```

Comment:
Does this mean that we only test one image? Can we considering testing at least 100-500 images to ensure the correctness?

Reply:
We always test with only one image in this tool. You could evaluate the model with this tool(https://github.com/open-mmlab/mmdetection/blob/master/tools/deployment/test.py) and test with a dataset.